src/Microsoft.Health.Fhir.Core/Features/Operations/Export/Models/ExportFileInfo.cs

```diff
@@ -51,5 +47,11 @@ protected ExportFileInfo()
 
         [JsonProperty(JobRecordProperties.CommitedBytes)]
         public long CommittedBytes { get; private set; }
+
+        public void Increment(int numberOfBytes)
```

Comment:
nit: `UpdateCommitCount(int numberOfBytes)` maybe?

Reply:
I renamed it to `IncrementCount(int numberOfBytes)`.